_integration-schemas/klaviyo/lists.md

```diff
@@ -0,0 +1,44 @@
+---
+tap: "klaviyo"
+version: "1.0"
+
+name: "lists"
+doc-link: ""
+singer-schema: "https://github.com/singer-io/tap-klaviyo/blob/master/tap_klaviyo/schemas/lists.json"
+description: ""
+
+replication-method: "Full Table"
+
+api-method:
+    name: ""
+    doc-link: "https://www.klaviyo.com/docs/api/metrics"
+
+attributes:
+  - name: "created"
+    type: "string"
+    description: ""
+  
+  - name: "id"
```

Comment:
This should have a `primary-key: true` attribute:

```suggestion
  - name: "id"
    type: "string"
    primary-key: true
    description: ""
```

It also needs to be first in the `attributes` list, so bump this up ahead of `created`.